agoradatatools/etl/transform.py

```diff
@@ -182,146 +164,105 @@ def transform_rna_seq_data(datasets: dict, adjusted_p_value_threshold: int):
+    target_list = datasets['target_list']
+    median_expression = datasets['median_expression']
+    druggability = datasets['druggability']
+    
+    # Modify the data before merging
+    
+    # All genes in this list should have 'isIGAP' = True when added to gene_metadata
+    igap['isIGAP'] = True 
+    
+    # Get the smallest p-value for each gene, to determine significance
     rna_change = rna_change.groupby('ensembl_gene_id')['adj_p_val'].agg('min').reset_index()
-
-    gene_metadata = pd.merge(left=gene_metadata,
-                             right=rna_change,
-                             how='left',
-                             on='ensembl_gene_id')
-    gene_metadata['adj_p_val'] = gene_metadata['adj_p_val'].fillna(-1)
-    gene_metadata['rna_brain_change_studied'] = gene_metadata.apply(
-        lambda row: False if row['adj_p_val'] == -1 else True, axis=1)
-    gene_metadata['rna_in_ad_brain_change'] = gene_metadata.apply(
-        lambda row: True if row['adj_p_val'] <= adjusted_p_value_threshold else False, axis=1)
-
-    gene_metadata = gene_metadata[
-        ['ensembl_gene_id', 'name', 'summary', 'alias', 'igap', 'symbol', 'eqtl', 'rna_in_ad_brain_change',
-         'rna_brain_change_studied']]
-
-    proteomics_concat = pd.concat([proteomics, proteomics_tmt])
     
+    # Get the smallest cor_pval for each protein, to determine significance
+    proteomics_concat = pd.concat([proteomics, proteomics_tmt])
     proteomics_concat = proteomics_concat.dropna(subset=['log2_fc', 'cor_pval', 'ci_lwr', 'ci_upr'])
     proteomics_concat = proteomics_concat.groupby('ensg')['cor_pval'].agg('min').reset_index()
-
-    gene_metadata = pd.merge(left=gene_metadata,
-                             right=proteomics_concat,
-                             how='left',
-                             left_on='ensembl_gene_id',
-                             right_on='ensg')
-    gene_metadata['cor_pval'] = gene_metadata['cor_pval'].fillna(-1)
-    gene_metadata['protein_brain_change_studied'] = gene_metadata.apply(
-        lambda row: False if row['cor_pval'] == -1 else True, axis=1)
-    gene_metadata['protein_in_ad_brain_change'] = gene_metadata.apply(
-        lambda row: True if row['cor_pval'] <= protein_level_threshold else False, axis=1)
-
-    gene_metadata = gene_metadata[
-        ['ensembl_gene_id', 'name', 'summary', 'symbol', 'alias', 'igap', 'eqtl', 'rna_in_ad_brain_change',
-         'rna_brain_change_studied', 'protein_in_ad_brain_change', 'protein_brain_change_studied']]
-    gene_metadata.rename(columns={'symbol': 'hgnc_symbol'}, inplace=True)
-
-    return gene_metadata
-
-
-def transform_gene_info(datasets: dict):
-
-    gene_metadata = datasets['gene_metadata']
-    target_list = datasets['target_list']
-    median_expression = datasets['median_expression']
-    druggability = datasets['druggability']
-
+    proteomics_concat.rename(columns = {'ensg': 'ensembl_gene_id'}, 
```

Comment:
we shouldn't be renaming columns in transforms, we should be using column_rename in config

Reply:
I will fix this -- I did this before realizing that column_rename would work without throwing errors on data sets that didn't have the requested columns.

Reply:
Fixed, this is now done as a `column_rename` instead of in code. (commit [1258ed40f91d6842e381cbaaac1eeef0d729c6cf](https://github.com/jaclynbeck-sage/agora-data-tools/commit/1258ed40f91d6842e381cbaaac1eeef0d729c6cf))